srcpkgs/nvidia340/template

```diff
@@ -34,14 +34,16 @@ fi
 do_extract() {
 	install -m755 ${XBPS_SRCDISTDIR}/${pkgname}-${version}/${_pkg}.run ${wrksrc}
 	cd ${wrksrc}
-	./${_pkg}.run --extract-only
-	rm -f ${_pkg}.run
+	ln -snf /usr/bin/bsdtar tar
```

Comment:
bsdtar does not need external utilities, unlike GNU tar.

Please get rid of bsdtar and its symlinks, and simply use "tar".

Reply:
I figured out why I had difficulties. When either `bsdtar` or `tar` is used inside of `depends`, no symlinks are created for `/usr/bin/tar`. Moving `tar` to `hostmakedepends` solves the missing symlink issue.

`xz` is called from their run script, however; it doesn't seem to mind having it missing... So I removed it.

Thanks,
-Joe